pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -4393,6 +4461,22 @@ public void ShowPeakAreaPeptideGraph()
             SynchronizeSummaryZooming();
         }
 
+        public void ShowPeakAreaRelativeAbundanceGraph()
+        {
+            ShowTotalTransitions();
+            Settings.Default.AreaGraphTypes.Insert(0, GraphTypeSummary.abundance);
+            ShowGraphPeakArea(true, GraphTypeSummary.abundance);
+            foreach (var summary in _listGraphPeakArea)
+            {
+                if (summary.Type == GraphTypeSummary.abundance)
+                {
+                    summary.Window = this;
+                }
+            }
+            UpdatePeakAreaGraph();
+            SynchronizeSummaryZooming();
```

Comment:
I can't see how this would apply to this graph. I don't think you could synchronize this with any other graph.